Drop useless header. (fixing the issue to the moved xmlrpcpp headers in
ros-kinetic)

--- src/robot_model.hh~	2016-10-04 02:24:44.000000000 +0200
+++ src/robot_model.hh	2017-04-04 17:43:38.779256884 +0200
@@ -5,7 +5,6 @@
 
 #include <sot-dynamic/dynamic.h>
 #include <dynamic-graph/linear-algebra.h>
-#include "XmlRpcValue.h"
 
 namespace dynamicgraph
 {
